pkg/plugins/platform/router.go

```diff
@@ -0,0 +1,92 @@
+package platform
```

Comment:
I agree there's more to this than what I suggested for router functionality here. I do think we should decide on a package for business functionality that is not platform-specific, and consolidate into that package. I suggested ddevapp since it's the most generically named, and we could make that the target moving forward. We can totally deal with this separate from this PR though.